path_config.go

```diff
@@ -267,8 +278,9 @@ func (b *backend) configDeleteOperation(ctx context.Context, req *logical.Reques
 }
 
 type config struct {
-	LDAP           *client.Config
-	PasswordPolicy string `json:"password_policy,omitempty"`
+	LDAP                         *client.Config
+	PasswordPolicy               string `json:"password_policy,omitempty"`
+	SkipStaticRoleImportRotation bool   `json:"skip_static_role_import_rotation"`
```

Comment:
It would get lengthy, but I wonder if adding some notion of `password` would make this more obvious.
`skip_static_role_import_password_rotation`. Maybe it's redundant.. thoughts?

Reply:
i don't mind adding it if people want it - i think rotation implies password in Vault contexts, so i don't think we _need_ it